apps/web/components/booking/BookingListItem.tsx

```diff
@@ -246,23 +246,26 @@ function BookingListItem(booking: BookingItemProps) {
       {/* NOTE: Should refactor this dialog component as is being rendered multiple times */}
       <Dialog open={rejectionDialogIsOpen} onOpenChange={setRejectionDialogIsOpen}>
         <DialogContent>
```

Comment:
NIT Dialog content actually accepts props to get this looking as per spec
<img width="800" alt="CleanShot 2023-02-09 at 15 21 15@2x" src="https://user-images.githubusercontent.com/55134778/217854941-3d0b3bf7-d6dd-4d33-9d9a-b0bf6b1bd1ce.png">